### DISCUSSION TITLE
Any built in report viewer control ?

### DISCUSSION THREAD

**Discussion by gurukalis** [unoplatform/uno#13784] [Q&A] [answered]
Is it support crystal report or rdlc ,or any builtin open source reportviewer control

**Comment by jeromelaban** [comment]
Thanks for the question! Could you specify which platforms are of interest to you?

**Comment by gurukalis** [comment]
reporting control means i could expecting like crystal report or rdlc

**Comment by gurukalis** [comment]
uno is cross platform right
we are planning to start some erp modules in new platform(uno)  is it have
all the possibilities )like print pos receipt

**Reply by jeromelaban** [reply]
Thanks. By which platforms to you want to target, I meant iOS, Android, WebAssembly, or other. Which ones are of interest to you?

**Reply by jeromelaban** [reply, marked as answer]
Thanks. Answering via email makes conversations difficult to follow (that's by design from github conversations), make sure to answer directly inside the thread so we can follow properly.

For WebAssembly, we're not aware of reporting tools that have been built for Uno at this time, but you can use existing javascript components and [embed them in your app](https://platform.uno/docs/articles/interop/wasm-javascript-1.html).

**Comment by gurukalis** [comment]
Webassembly
